test/DartTest.ts

```diff
@@ -107,6 +108,24 @@ describe('transpile to dart', () => {
     });
   });
 
+  describe('enums', () => {
+    it('should support basic enum declaration', () => {
+      expectTranslate('enum Color { Red, Green, Blue }')
+          .to.equal(' enum Color { Red , Green , Blue }');
+    });
+    it('does not support empty enum', () => {
+      chai.expect(() => translateSource('enum Empty { }'))
+          .to.throw('empty enums are not supported');
+    });
+    it('does not support enum with initializer', () => {
+      chai.expect(() => translateSource('enum Color { Red = 1, Green, Blue = 4 }'))
+          .to.throw('enum initializers are not supported');
+    });
+    it('should support switch over enum', () => {
+      expectTranslate('switch(c) { case Color.Red: break; default: break; }')
```

Comment:
Right. (This test just made me feel better that a case label could be written as, e.g., `Color.Red`. I didn't know for sure until I wrote the test.)